### PR TITLE
Incorporated DB setup updates

### PR DESCRIPTION
**Pull request by cwarnermm**

### CODE REVIEW

**Review by nab-77 — APPROVED** [review]
All good with the changes but I keep coming back to how much we need to document re the service stack. Here this is how to setup pgsql on RHEL (and its nuances). We just need a pgsql to connect to (and there for only need to define the perms needed for the mm dm user).

**Comment by cwarnermm** [comment]
@nab-77 @coltoneshaw - My latest commit to this docs PR attempts to simplify the RHEL install docs page and align it with our new approach (parallel to Ubuntu and tarball). 

Under the covers, I've turned the common steps into reusable components we can share between RHEL and tarball (since a signed package for RHEL isn't yet available), which reduces/eliminates content duplication risk.

@coltoneshaw - This latest commit also takes your updates via https://github.com/mattermost/docs/pull/6576 into account. Please review this PR to ensure that I've included all of your updates specific to RHEL installs. Right now, I believe this RHEL page is very v8 specific and may need additional updates to capture relevant v7 details. Please let me know.

@nab-77 - This latest commit includes updates to the database install page to account for @coltoneshaw's corrections specific to RHEL. I agree with your feedback that this page was too instruction-heavy and needed to be brought in line with our other simplified install pages. All feedback is welcome.

To review the changes in context using the generated preview, please visit:
- [Prepare your Mattermost database](http://mattermost-docs-preview-pulls.s3-website-us-east-1.amazonaws.com/6559/install/prepare-mattermost-database.html)
- [Install Mattermost on RHEL](http://mattermost-docs-preview-pulls.s3-website-us-east-1.amazonaws.com/6559/install/install-rhel-8.html)
- [Install Mattermost Server using the tarball](http://mattermost-docs-preview-pulls.s3-website-us-east-1.amazonaws.com/6559/install/install-tar.html) -- review only to ensure that reusable components display as intended.

File: source/install/prepare-mattermost-database.rst [hunk 1]
```diff
@@ -124,16 +188,16 @@ To set up a PostgreSQL database for use by the Mattermost server:
 
       .. code-block:: none
         :class: mm-code-block 
-    
+
           psql --dbname=mattermost --username=mmuser --password
 
   .. tab:: Multi-server deployment
```
Comment (nab-77) [inline source/install/prepare-mattermost-database.rst:194]: For me this not a cause of multi-server deployment, rather better phrased as "db on the same server as mattermost" or "db on a remote system".

0/5 on the terms but the concept is there.

File: source/install/install-rhel-8.rst [hunk 2]
```diff
@@ -3,169 +3,99 @@
 Install Mattermost on RHEL
```
Comment (nab-77) [inline source/install/install-rhel-8.rst:3]: Just a note that the code blocks in-line with text do not seem to have the style applied (i.e. the colour)

**Review by nab-77 — CHANGES_REQUESTED** [review]

File: source/install/prepare-mattermost-database.rst (hunk 1 above)
Comment (cwarnermm) [reply source/install/prepare-mattermost-database.rst:194]: @coltoneshaw - What terminology do you tend to hear from customers when differentiating between a deployment where all components are on a single server versus components spread across servers?

**Comment by cwarnermm** [comment]
@nab-77 @coltoneshaw - This docs PR is ready for your review.

File: source/install/install-rhel-8.rst (hunk 2 above)
Comment (nab-77) [reply source/install/install-rhel-8.rst:3]: On https://docs.mattermost.com/install/install-tar.html you can see the in-line code is the same blue as the code boxes.

Comment (cwarnermm) [reply source/install/install-rhel-8.rst:3]: Thank you!

**Review by nab-77 — APPROVED** [review]

File: source/install/prepare-mattermost-database.rst (hunk 1 above)
Comment (coltoneshaw) [reply source/install/prepare-mattermost-database.rst]: Oh, I'd +1 neil on this one. Although, both are technically correct. I'd lean towards `DB on another server`

**Pull request merged**